src/swapd/runtime.rs

```diff
@@ -711,15 +712,6 @@ impl SyncerState {
             spend_key,
             address,
         });
-        let from_height = match self.lock_tx_confs {
-            Some(Request::SyncerEvent(Event::TransactionConfirmations(
-                TransactionConfirmations {
-                    confirmations: Some(tx_confs),
-                    ..
-                },
-            ))) => Some(self.monero_height - tx_confs as u64),
-            _ => None,
-        };
```

Comment:
lock_tx_confs contains the bitcoin lock confs, in btc blocks, not xmr blocks

Reply:
Damn, I still find the `lock` terminology confusing, but after reading it again, it's more clear to me now. Still, the confirmations (so the block height of the sent transaction) and not the current block height should be taken into account.

Reply:
i agree. that special lock_tx_confs was created ad-hoc to replay the last syncer event when receiving txs from wallet (and updating swapd state), so that if there is some action to trigger in swapd, it triggers right after having the tx available (it becomes reactive)